src/Resource/Item.php

```diff
@@ -172,14 +112,30 @@ public function getType(): string
         return $this->type;
     }
 
+    /**
+     * @param string $type
+     */
+    public function setType(string $type): void
+    {
+        $this->type = $type;
+    }
+
     /**
      * @return bool|null
      */
-    public function isStackable(): ?bool
+    public function getStackable(): ?bool
     {
         return $this->stackable;
     }
 
+    /**
+     * @param bool|null $stackable
+     */
+    public function setStackable(?bool $stackable): void
+    {
+        $this->stackable = $stackable;
+    }
```

Comment:
@mwansinck kunnen we dit niet veel beter eruit laten voor nu? Roept voor de pakket klanten alleen maar vragen op en is alleen van toepassing bij de Pallets.

Reply:
Opzich geen probleem om het uit te SDK te laten natuurlijk, maar momenteel is die ook al sinds dat de API bestaat schrijfbaar daar. Denk niet dat die gebruikt wordt maar dan zou de API op het gebied van items inschieten wel afwijken van wat de SDK kan.